interface-definitions/container.xml.in

```diff
@@ -104,11 +104,28 @@
               </leafNode>
             </children>
           </tagNode>
+          <leafNode name="entrypoint">
+            <properties>
+              <help>List of entrypoints</help>
+              <multi/>
```

Comment:
Is it support multiple entrypoints?

Reply:
It's more like a entrypoint arguments.
I checked the [documents](https://docs.podman.io/en/latest/markdown/podman-run.1.html#entrypoint-command-command-arg1) and found that it's specified as a json formatted array.

For example

```json
["bin/sh", "-c", "..."]
```

Reply:
For example

```
set container name test entrypoint 'abc.example.com'
set container name test entrypoint /bin/sh
set container name test entrypoint -c
set container name test entrypoint '--count 10'
set container name test entrypoint ping
```
How will order such args? Maybe wrong example, but I'll try eplain what do I mean
We get
```
["abc.example.com", "bin/sh", "-c", "--count 10", "ping"]
```
It seems expecting something like this order
```
["bin/sh",  "-c", "ping", "--count 10",  "abc.example.com"]
```

Reply:
That's correct, I think the best way to set theses is to first delete, then set.

```shell
delete container name test entrypoint
```

```shell
set container name test entrypoint /bin/sh
set container name test entrypoint -c
set container name test entrypoint '&quot;--count 10'
set container name test entrypoint ping
set container name test entrypoint 'abc.example.com&quot;'
```

Reply:
It will be sorted after saving and reboot :)
So in my opinion it should be single leafNode, not multi
```
set container name test entrypoint '/bin/sh -c ping --count 10 abc.example.com'
```
Then you split it internally to list ["/bin/sh",  "-c", "ping", "--count 10",  "abc.example.com"] if it is required

Reply:
Thanks for the catch, I'll made the change for the arguments as well.

Reply:
I have tested and publish a new commit. Please take another look.